gogettr/cli.py

```diff
@@ -163,3 +163,11 @@ def comments(post_id: str, max: int = None):
     """Pull comments on a specific post."""
     for comment in client.comments(post_id=post_id, max=max):
         print(json.dumps(comment))
+
+
+@cli.command()
+@click.argument("username")
+def check_username(username: str):
```

Comment:
I'm not sure if we should make the CLI-facing command `check-username` while the internal name is `RegistrationStatus`. Could we rename this command to `registered`?

Reply:
Right, better.